python-sdk/src/astro/sql/operators/base_decorator.py

```diff
@@ -1,24 +1,30 @@
 from __future__ import annotations
 
 import inspect
-from typing import Any, cast
+from typing import Any, Sequence, cast
 
+import jinja2
 import pandas as pd
 from airflow.decorators.base import DecoratedOperator
 from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
 from astro.airflow.datasets import kwargs_with_datasets
 from astro.databases import create_database
 from astro.databases.base import BaseDatabase
 from astro.sql.operators.upstream_task_mixin import UpstreamTaskMixin
 from astro.sql.table import BaseTable, Table
 from astro.utils.table import find_first_table
 from astro.utils.typing_compat import Context
+from jinja2 import DebugUndefined
 from sqlalchemy.sql.functions import Function
 
 
 class BaseSQLDecoratedOperator(UpstreamTaskMixin, DecoratedOperator):
     """Handles all decorator classes that can return a SQL function"""
 
+    template_fields: Sequence[str] = ("sql", "parameters", "op_args", "op_kwargs")
```

Comment:
Have you tested on airflow to see if the SQL now shows up in the "rendered" section?

Reply:
Yes. It does.